plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/animators/PuckPulsingAnimator.kt

```diff
@@ -19,19 +22,35 @@ internal class PuckPulsingAnimator : PuckAnimator<Double>(Evaluators.DOUBLE) {
   }
 
   fun animateInfinite() {
-    animate(0.0, maxRadius)
+    if (maxRadius <= 0.0) {
```

Comment:
I'd replace with `==` check, passing let's say -2 is more of an error to me

Reply:
Currently `maxRadius` can be `-1.0`, this is a safeguard so maxRadius can't be less than 0.0. Also in opacity calculation we divide using `maxRadius` , so i guess its better to have this condition.